src/api/procedures/setAssetRequirements.ts

```diff
@@ -26,6 +26,28 @@ export type Params = SetAssetRequirementsParams & {
   ticker: string;
 };
 
+/**
+ * @hidden
+ */
+function identityTransformation(
```

Comment:
You should create a `conditionLikeToCondition(conditionLike: ConditionLike, context: Context): Condition` util in the conversion file and use that